Assets/UI/PartialScreens/citystates.lua

```diff
@@ -1350,13 +1336,42 @@ function ViewCityState( iPlayer:number )
   end
 end
 
+-- ===========================================================================
+function OnSingleViewStackSizeChanged()
+  Controls.ReportTabContainer:SetParentRelativeSizeY(REPORT_CONTAINER_SIZE_PADDING - Controls.SingleViewStack:GetSizeY() - Controls.SingleViewStack:GetOffsetY());
+end
+  
+-- ===========================================================================
+function AddInfluenceRow(cityStateID:number, playerID:number, influence:number, largestInfluence:number)
+  local kItem :table = m_InfluenceRowIM:GetInstance();
+
+  local localPlayerID:number = Game.GetLocalPlayer();
+  local pLocalPlayerDiplomacy:table = Players[localPlayerID]:GetDiplomacy();
+  
+  local pPlayerConfig :table = PlayerConfigurations[playerID];
+  
+  local civName :string = "LOCAL_CITY_STATES_UNKNOWN";
+  if (localPlayerID == playerID) then
+    civName = Locale.Lookup(pPlayerConfig:GetPlayerName()) .. " (" .. Locale.Lookup("LOC_CITY_STATES_YOU") .. ")";
+  elseif (pLocalPlayerDiplomacy:HasMet(playerID)) then
+    civName = pPlayerConfig:GetPlayerName();
+  end
+  kItem.CityName:SetText( Locale.ToUpper(civName) );
+
+  kItem.AmountBar:SetPercent( influence / largestInfluence);
+  kItem.Amount:SetText( tostring(influence) );
+  
+  return kItem;
+end
+
 -- ===========================================================================
 function ViewRelationships( kCityState:table )
 	-- Setup the buttons and what (sub) areas are shown/hidden.
 	Controls.EnvoysSentButton:SetSelected( false );
 	Controls.InfluencedByButton:SetSelected( false );
 	Controls.QuestsButton:SetSelected( false );
-	Controls.RelationshipsButton:SetSelected( true );
+  Controls.RelationshipsButton:SetSelected( true );
+  Controls.SingleViewStack:RegisterSizeChanged( OnSingleViewStackSizeChanged );
```

Comment:
This `Controls.SingleViewStack:RegisterSizeChanged( OnSingleViewStackSizeChanged );` should not be here. You misplaced it, it's in `Initialize()`